phoenix-core/src/main/java/org/apache/phoenix/monitoring/GlobalMetricRegistriesAdapter.java

```diff
@@ -109,20 +109,20 @@ private void snapshotAllMetrics(MetricRegistry metricRegistry,
             Map<String, Metric> metrics = metricRegistry.getMetrics();
             Iterator iterator = metrics.entrySet().iterator();
 
-            while (iterator.hasNext()) {
-                Entry<String, Metric> e = (Entry) iterator.next();
+            while(iterator.hasNext()) {
+                Entry<String, Metric> e = (Entry)iterator.next();
```

Comment:
tiny nit: the extra space between casted class and variable aids readability (and removing the space it makes this patch larger). I believe our checkstyle wants the space too, iirc

Reply:
Thanks, fixed.